### PR TITLE
OCPQE-17380: create dedicated installation ci yaml file

### PR DESCRIPTION
**Pull request by jianlinliu**

### CODE REVIEW

**Comment by jianlinliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-installation-nightly-aws-ipi-preserve-ign-arm-f14

**Comment by jianlinliu** [comment]
pj [job](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/44502/rehearse-44502-periodic-ci-openshift-openshift-tests-private-release-4.14-installation-nightly-aws-ipi-preserve-ign-arm-f14/1714525779821334528/artifacts/aws-ipi-preserve-ign-arm-f14/cucushift-installer-check-aws-bootstrap-ignition/build-log.txt) get passed:

**Comment by jianlinliu** [comment]
cc @jhou1 @yunjiang29 @gpei @liangxia to review.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__installation-nightly.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      OCP_ARCH: arm64
+    test:
+    - ref: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-aws-ipi-preserve-bootstrap-ign
+zz_generated_metadata:
+  branch: release-4.15
+  org: openshift
+  repo: openshift-tests-private
```
Comment (yunjiang29) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__installation-nightly.yaml:75]: I thought that we could have a separate repo, is it possible? `openshift-openshift-tests-private-release-4.15-periodics.yaml` will become huge.

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__installation-nightly.yaml:75]: Some doubt:
1. can we freely change any repo name here?
2. once changed, does RP classifier still upload the result to RP?

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__installation-nightly.yaml:75]: Perhaps we can reuse the verification-tests repo ?

Comment (yunjiang29) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__installation-nightly.yaml:75]: > 2\. once changed, does RP classifier still upload the result to RP?

@JianLi-RH can you help confirm?

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__installation-nightly.yaml:75]: > Perhaps we can reuse the verification-tests repo ?

That is okay for me. Then we only need to confirm the 2nd question.

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__installation-nightly.yaml:75]: Changes needed for classifier job to get this project included.

**Pull request closed without merging**